addons/mail/static/src/models/chat_window/chat_window.js

```diff
@@ -178,6 +178,50 @@ function factory(dependencies) {
             return this.env._t("New message");
         }
 
+        /**
+         * @returns {boolean}
+         * @private
+         */
+        _computeIsVisible() {
+            if (!this.managerAllOrderedVisible) {
```

Comment:
Did you have the issue? x2many should be the empty list by default, this should never be empty.

Reply:
Nop, kind of an habbit to code in the defensive way